bliss/models/prior.py

```diff
@@ -87,10 +84,9 @@ def __init__(
 
         self.prob_galaxy = float(prob_galaxy)
         if prob_galaxy > 0.0:
-            latents = get_galaxy_latents(latents_file, n_latent_batches, autoencoder_ckpt)
+            self.vae = galaxy_net.OneCenteredGalaxyAE.load_from_checkpoint(autoencoder_ckpt)
```

Comment:
at some point we should change the class name and checkpoint names from AE to VAE :)

Reply:
I agree - I opened up a separate issue since I want to update the names of all the modules in the "galaxy_net.py" file